yamljs/yamljs.d.ts

```diff
@@ -1,9 +1,9 @@
-// Type definitions for yamljs 0.2.1
+// Type definitions for yamljs 0.2.3
 // Project: https://github.com/jeremyfa/yaml.js
 // Definitions by: Tim Jonischkat <http://www.tim-jonischkat.de>
 // Definitions: https://github.com/borisyankov/DefinitelyTyped
 
-declare module "yamljs" {
```

Comment:
please keep this line. use this pattern https://github.com/borisyankov/DefinitelyTyped/blob/master/jszip/jszip.d.ts#L189-L191